capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Heading, Badge, Button, HStack, Image, VStack,
+} from '@chakra-ui/react';
+import { useState, useEffect } from 'react';
+
+const API_KEY = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
+
+function getPhotoReference(restaurant) {
+  return restaurant.photos[0].photo_reference;
+}
+
+export default function ExperienceInfo({ restaurants }) {
+  const [indexRestaurant, setIndexRestaurant] = useState(0);
+  const [currentRestaurant, setCurrentRestaurant] = useState({});
+  const [photoReference, setPhotoReference] = useState('');
```

Comment:
These seem like three pieces of tightly related state. Is there a way we could use a single piece of state, and have the rest of the values derived from it? 

This goes back to the discussion we had on the earlier store project: https://github.com/CarLosVegga/tdd-student-store-starter/pull/1#discussion_r910381346.